draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1122,6 +1127,32 @@ frames, which would waste some bandwidth.
 Application designers are advised to take these considerations into account when
 selecting and configuring a QUIC stack for use with RoQ.
 
+# Directions for Future work {#futures}
+
+This specification represents considerable work and discussion within the IETF, and describes RoQ in sufficient detail that an implementer can build a RoQ application, but we recognize that additional work is likely, after we have sufficient experience with RoQ to guide that work. Possible directions would include
+
+* Better guidance on transport for RTCP (for example, when to use QUIC streams vs. QUIC datagrams for RTCP transport).
+
+* Better guidance on the use of realtime-friendly congestion control algorithms (for example, Copa {{Copa}}, L4S {{?RFC9330}}, etc.).
+
+* Better guidelines and recommendations for congestion control and rate adaptation for multiple RoQ flows (whether streams or datagrams).
+
+* Possible guidelines for connection sharing between RoQ and non-RoQ flows.
```

Comment:
```suggestion
* Possible guidelines for connection sharing between RoQ and non-RoQ flows, including guidance on the use of congestion control, scheduling, prioritization, and guidance on which ALPNs to use.
```

Reply:
This is good in theory, but in practice, 

> * Possible guidelines for connection sharing between RoQ and non-RoQ flows, including guidance on the use of congestion control, scheduling, prioritization, and guidance on which ALPNs to use.

overlaps with 

> * Possible guidance for connection sharing between RoQ and non-RoQ flows.

and 

> * Guidance for using RoQ with QUIC extensions mentioned in {{optional-extensions}} for using Interactive Connectivity Establishment (ICE) {{?RFC8445}} with QUIC, including guidance for coexistence between QUIC connections established using ALPNs and QUIC connections established using ICE.

I'm going to use almost every word you suggested - I'll propose using them in different places. :upside_down_face:

(which OUGHT to include at least congestion control, scheduling, prioritization).

I'll also use "guidance" consistently (rather than a mixture of "guidance", "guidelines and recommendations", and "guidelines", which I intended to all mean the same thing).